src/post/post.service.ts

```diff
@@ -359,7 +371,7 @@ export class PostService {
                 this.on('likes.post_id', 'posts_with_source.id').andOnVal(
                     'likes.account_id',
                     '=',
-                    accountId.toString(),
+                    defaultAccountId.toString(),
```

Comment:
Earlier this query was used to get profile post of self account so we were checking if the post is liked/reposted by the account. Updated this to check if the post is liked/reposted by default account.

Reply:
This was a query change so I did it separately [here](https://github.com/TryGhost/ActivityPub/pull/488). 

> What is accountId now being used for

It is used to get the posts for that account. Where as the `defaultAccountId` is used to check if the post is liked/reposted by the default/current user.